frontend/src/components/BookingCard.tsx

```diff
@@ -64,20 +65,10 @@ const Icon = styled.img`
   padding: 20px;
 `
 
-export default function BookingCard() {
+export default function BookingCard({ bookings }: { bookings: Booking[] }) {
   const dispatch = useDispatch()
   const history = useHistory()
-  const params = useParams<{ facilityID: string }>()
-  const {
-    ViewStartDate,
-    ViewEndDate,
-    createSuccess,
-    createFailure,
-    isLoading,
-    facilityBookings,
-    selectedFacilityName,
-    selectedFacilityId,
-  } = useSelector((state: RootState) => state.facilityBooking)
+  const { ViewStartDate, facilityBookings } = useSelector((state: RootState) => state.facilityBooking)
```

Comment:
the start date and facility bookings are taken from the store? is this used or appropriate? bc the info are now passed into this component instead of being taken from the store